kbase-extension/static/kbase/js/common/cellComponents/actionButtons.js

```diff
@@ -70,40 +68,34 @@ define([
                     label: button.label
                 });
             });
-            this.bus.on('actionButton', (message) => {
-                const action = message.data.action;
-                this.runAction(action);
+            bus.on('actionButton', (message) => {
+                const action = message.data;
+                runAction(action);
             });
 
             return div({
-                class: `${cssCellType}-action-button__list btn-group`
+                class: 'btn-action__list btn-group'
             }, buttonList);
         }
 
-        /**
-         *
-         * @param {object} newState
-         *  - name: action button to show
-         *  - enabled: truthy if should be enabled
-         */
-        setState(newState) {
-            this.state = newState;
-            for (const btnName of Object.keys(this.actionButtons.availableButtons)) {
-                this.ui.hideButton(btnName);
+        function setState(newState) {
+            let state = newState;
+            for (const btnName of Object.keys(actionButtons.availableButtons)) {
+                ui.hideButton(btnName);
             }
-            this.ui.showButton(this.state.name);
-            this.state.enabled ? this.ui.enableButton(this.state.name) : this.ui.disableButton(this.state.name);
-        }
-
-        start() {
-
-        }
-
-        stop() {
-
+            ui.showButton(state.name);
+            state.disabled ? ui.disableButton(state.name): ui.enableButton(state.name);
```

Comment:
@briehl This was causing the enable bug that you found.

Reply:
It was `state.disable` and I changed it to `state.disabled` :)

Reply:
That fixed it. Thanks!